src/rgw/librgw.cc

```diff
@@ -227,7 +227,8 @@ namespace rgw {
     rgw_env.set("HTTP_HOST", "");
 
     /* XXX and -then- bloat up req_state with string copies from it */
-    struct req_state *s = req->get_state();
```

Comment:
hmm;  I guess this would have been a nullptr

Reply:
Yeah.